version.php

```diff
@@ -25,8 +25,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2025011400;      // The current plugin version (Date: YYYYMMDDXX).
-$plugin->release   = 2025011400;      // Same as version.
+$plugin->version   = 2025011401;      // The current plugin version (Date: YYYYMMDDXX).
+$plugin->release   = 2025011401;      // Same as version.
```

Comment:
With the current stable branch it should be a proper version bump (current date)